engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java

```diff
@@ -1838,7 +1838,7 @@ long getEndWindowEmitTimestamp(long windowId, PTOperator oper)
 
   public long updateOperatorLatency(PTOperator oper, UpdateOperatorLatencyContext ctx)
   {
-    if (!oper.getInputs().isEmpty()) {
+    if (!oper.getInputs().isEmpty() && oper.stats.currentWindowId.get() > 0) {
```

Comment:
Please add unit test. StreamingContainerManagerTest.testLatency() fails intermittently, so it does not cover the change.